vta/include/vta/driver.h

```diff
@@ -118,6 +118,22 @@ void VTAMemFree(void* buf);
  */
 vta_phy_addr_t VTAMemGetPhyAddr(void* buf);
 
+/*!
+ * \brief Performs a copy operation from host memory to buffer allocated with VTAMemAlloc.
+ * \param dst The desination buffer in FPGA-readable memory. Has to be allocated with VTAMemAlloc().
+ * \param src The source buffer in host memory.
+ * \param size Size of the region in Bytes.
+ */
+void VTAMemMoveToBuffer(void* dst, const void* src, size_t size);
```

Comment:
Normal API name will name it as Copy. I would rename it as VTAMemCopyToHost

Reply:
this is a good naming suggestion; I'll make the change